test/token/ERC20/ERC20.behavior.js

```diff
@@ -240,14 +225,6 @@ function shouldBehaveLikeERC20Transfer (errorPrefix, from, to, balance, transfer
       });
     });
   });
-
-  describe('when the recipient is the zero address', function () {
```

Comment:
Since everything goes through the transfer, it cannot know if its coming from mint or burn, so `transfer` has a check to ensure that `from` and `to` cannot be the zero address, but if a user sends a mint to address zero it would received both `from` and `to` as the zero address. There is a test for that case on [ln 206](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/27c7090480c2dc26f361def7b4a62960a591f55b/test/token/ERC20/ERC20.test.js#L206)

Reply:
Line 206 is testing that you cannot mint to address 0. That is a valid test, but its separate from this. This tests that you cannot call transfer to the address 0. 

Tests are checking for behavior correctness. Its ok (and I think its needed) to have multiple tests that are resolved by the same require in different context. 

I think this test should have been kept.

Reply:
I wonder if that is the cause of the coverage decrease.